R/tree.R

```diff
@@ -57,11 +57,8 @@
 #'                 here("src/FastTree")
 #' }
 convertFA2Tree <- function(fa_path = here("data/alns/pspa_snf7.fa"),
```

Comment:
If these default/example files don't exist, we could remove them. Otherwise, placeholders that work might be a good idea!